easybuild/easyconfigs/m/MiXCR/MiXCR-4.6.0-Java-17.eb

```
@@ -0,0 +1,28 @@
+easyblock = 'Tarball'
+
+name = 'MiXCR'
+version = '4.6.0'
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = 'https://milaboratory.com/software/mixcr'
+description = """MiXCR is a universal software for fast and accurate extraction of T- and B- cell receptor repertoires
+from any type of sequencing data."""
+
+toolchain = SYSTEM
+
+source_urls = ['https://github.com/milaboratory/mixcr/releases/download/v%(version)s/']
+sources = [SOURCELOWER_ZIP]
+checksums = ['05db1276951a2e656d0a7bf4e2b1fff326733a5f961a9d4829be139852fabe13']
+
+dependencies = [('Java', '17')]
+
+sanity_check_paths = {
+    'files': ['mixcr', 'mixcr.jar'],
+    'dirs': [],
+}
+
+sanity_check_commands = ["command -v mixcr"]
```

@pavelToman it's better to run `mixcr` directly. the following command does not need a license:
```suggestion
sanity_check_commands = ["mixcr -v"]
```